modules/asr/Cyberon/FlashReadData.c

```diff
@@ -0,0 +1,56 @@
+// Copyright 2021-2022 XMOS LIMITED.
+// This Software is subject to the terms of the XMOS Public Licence: Version 1.
+
+/* System headers */
+#include <stdint.h>
+#include <xs1.h>
+#include <xcore/assert.h>
+
+/* FreeRTOS headers */
+#include "FreeRTOS.h"
+#include "task.h"
+
+/* Library headers */
+#include "asr.h"
+#include "rtos_qspi_flash.h"
+
+/* App headers */
+#include "platform/driver_instances.h"
+#include "FlashReadData.h"
+
+#define DBG_TRACE                 // Output log by DbgTrace(), if not defined only output by rtos_printf.
+#include "DbgTrace.h"
+
+#if 1
```

Comment:
Please include a comment stating why this file contains two different implementations of `FlashReadData()` and why the selected one is in use.

Reply:
I removed the second implementation because:
 - it is said in the comments that it is slower
 - the first implementation is used in the Sensory ASR too